src/components/ui-shell/ui-shell--header.hbs

```diff
@@ -0,0 +1,51 @@
+<header class="bx--platform-header">
+  <button class="bx--platform-header__menu-trigger bx--platform-header__action" title="Open menu">
+    <svg width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" aria-hidden="true">
+      <path d="M8.24 25.14L7 26.67a14 14 0 0 0 4.18 2.44l.68-1.88a12 12 0 0 1-3.62-2.09zm-4.05-7.07l-2 .35A13.89 13.89 0 0 0 3.86 23l1.73-1a11.9 11.9 0 0 1-1.4-3.93zm7.63-13.31l-.68-1.88A14 14 0 0 0 7 5.33l1.24 1.53a12 12 0 0 1 3.58-2.1zM5.59 10L3.86 9a13.89 13.89 0 0 0-1.64 4.54l2 .35A11.9 11.9 0 0 1 5.59 10zM16 2v2a12 12 0 0 1 0 24v2a14 14 0 0 0 0-28z" />
+    </svg>
+  </button>
+  <a class="bx--platform-header__name" href="#" title="Platform home">
+    IBM [Platform]
+  </a>
+  <nav class="bx--platform-header__nav">
+    <ul class="bx--platform-header__links">
+      <li class="bx--platform-header__link-item">
```

Comment:
Great to see Handlebars `{{#each}}` directive used in this file - Ref: https://github.com/IBM/carbon-components/pull/997

Reply:
Can definitely dive into the configuration, maybe I'll add that in another PR when we have sub-menus for the links? That feels complex enough to warrant starting to add it in.

Reply:
Yeah doing that in subsequent change will be good - Thanks @joshblack!